### PR TITLE
feat: add agenda component

### LINKED ISSUE #162: Implement Agenda Page
Speakers for London have been selected, and we need to start working on the agenda page for AACoT'23 London.

@Mayaleeeee [made some awesome designs for this](https://www.figma.com/proto/sS5ofLRtuFqDGFvlNBurns/AACoT?type=design&node-id=817-884&t=wvHkqwlhQmI6K4BO-1&scaling=min-zoom&page-id=0%3A1&starting-point-node-id=817%3A720&mode=design), we just need to in cooperate it to the conference website.

cc @AceTheCreator @thulieblack

### PR DESCRIPTION
**Pull request by AceTheCreator**
Implemented the agenda component and rendered the conference agenda for the London Edition. 

cc @thulieblack @derberg @Mayaleeeee 

resolves #162

### CODE REVIEW

**Comment by thulieblack** [comment]
I have a request, is there a way we can the talk abstract/summary of what the talk will be all about (this can be a dropdown column where someone can click on the title and get the details) @AceTheCreator

File: components/Agenda/agenda.js [hunk 1]
```diff
@@ -2,20 +2,63 @@ import React from 'react'
...
-		<div className='container flex flex-col justify-center items-center'>
-			<Heading className='text-[30px] text-white'>
+	  <div className=''>
+		  <div className='flex flex-col justify-center items-center'>
+			  <Heading className='text-[30px] text-white'>
 				Agenda
 			</Heading>
-			<Paragraph className='mt-[16px]'>
+			<Paragraph className='mt-[16px] text-center'>
 				The conference will commence at 8:00am BST (UTC+1)
```
Comment (thulieblack) [inline components/Agenda/agenda.js:14]:
```suggestion
				The conference will commence at 9:00am BST (UTC+1)
```

**Comment by AceTheCreator** [comment]
> I have a request, is there a way we can the talk abstract/summary of what the talk will be all about (this can be a dropdown column where someone can click on the title and get the details) @AceTheCreator

yea, it's very possible @thulieblack

**Comment by thulieblack** [comment]
Awesome, because it will be helpful to have it, especially when we have people viewing what the talks will be all about. 

We can merge this so long while we work on that. thoughts @derberg

**Comment by Mayaleeeee** [comment]
Looks good @AceTheCreator.
Thank you.

**Comment by derberg** [comment]
@AceTheCreator @thulieblack @Mayaleeeee please do not wait for my approval. I'm not a codeowner here. It's all yours. If I have an opinion I can share it after merging as any other community member.

anyway, agenda looks great, I love the idea to show speaker profile pic next to the title of the talk, great stuff.

go ahead and merge whenever you want. @AceTheCreator I suggest you setup CODEOWNERS file in a separate PR

**Comment by AceTheCreator** [comment]
> go ahead and merge whenever you want. @AceTheCreator I suggest you setup CODEOWNERS file in a separate PR

I'll do just that :)

**Pull request merged**